app/models/user.rb

```diff
@@ -11,10 +11,11 @@ class User < ApplicationRecord
                     length: { minimum: 6, maximum: 100 },
                     format: { with: URI::MailTo::EMAIL_REGEXP }
   validates :password, presence: true,
+                       on: :create,
```

Comment:
it's better to use `unless: :skip_password` and add `skip_password` `attr_reader` and use it